Makefile

```
@@ -0,0 +1,103 @@
+# /!\ /!\ /!\ /!\ /!\ /!\ /!\ DISCLAIMER /!\ /!\ /!\ /!\ /!\ /!\ /!\ /!\
+#
+# This Makefile is only meant to be used for DEVELOPMENT purpose as we are
+# changing the user id that will run in the container.
+#
+# PLEASE DO NOT USE IT FOR YOUR CI/PRODUCTION/WHATEVER...
+#
+# /!\ /!\ /!\ /!\ /!\ /!\ /!\ /!\ /!\ /!\ /!\ /!\ /!\ /!\ /!\ /!\ /!\ /!\
+#
+# Note to developpers:
+#
+# While editing this file, please respect the following statements:
+#
+# 1. Every variable should be defined in the ad hoc VARIABLES section with a
+#    relevant subsection
+# 2. Every new rule should be defined in the ad hoc RULES section with a
+#    relevant subsection depending on the targeted service
+# 3. Rules should be sorted alphabetically within their section
+# 4. When a rule has multiple dependencies, you should:
+#    - duplicate the rule name to add the help string (if required)
+#    - write one dependency per line to increase readability and diffs
+# 5. .PHONY rule statement should be written after the corresponding rule
+# ==============================================================================
+# VARIABLES
+
+# -- Docker
+# Get the current user ID to use for docker run and docker exec commands
+DOCKER_UID           = $(shell id -u)
+DOCKER_GID           = $(shell id -g)
+DOCKER_USER          = $(DOCKER_UID):$(DOCKER_GID)
+
+COMPOSE              = DOCKER_USER=$(DOCKER_USER) docker-compose
+COMPOSE_RUN          = $(COMPOSE) run --rm
+COMPOSE_RUN_APP      = $(COMPOSE_RUN) app
+COMPOSE_RUN_NODE      = $(COMPOSE_RUN) -e HOME="/tmp" app
+
+YARN                 = $(COMPOSE_RUN_NODE) yarn
+
+# ==============================================================================
+# RULES
+
+default: help
+
+# -- Test suite
+
+stress: ## run stress test
+	@$(COMPOSE_RUN_APP) ./cli.js stress -v
+.PHONY: stress
+
+list-meetings: ## list meetings running on the BBB server
+	@$(COMPOSE_RUN_APP) ./cli.js list-meetings -v
+.PHONY: list-meetings
+
+
+# -- Project bootstrap
+
+.env:
+	@if [ ! -f .env ]; then cp .env.default .env ; echo ".env file generated successfully. Please edit it to set BBB_URL, BBB_SECRET and BBB_MEETING_ID" ; fi
```

Why this `if` statement ? Make will not exute this task if the file already exists

```suggestion
	cp .env.default .env
	@ echo ".env file generated successfully. Please edit it to set BBB_URL, BBB_SECRET and BBB_MEETING_ID" 
```

---

yes, I forgot that :smile: